Formula/hatari.rb

```diff
@@ -23,9 +23,14 @@ class Hatari < Formula
 
   depends_on "cmake" => :build
   depends_on "libpng"
-  depends_on "python@3.10"
   depends_on "sdl2"
 
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "readline"
+  end
```

Comment:
These are already in Linux dependency tree but let's make them direct since they are linked to enable features:
```
  - readline :	found, enables history/completion in the debugger
  - zlib :	found, allows to use zip/gz files directly
```

Note that macOS existing bottle linked to `readline` via Python. New bottle will link to system `libedit` instead, so we probably want new bottles to avoid missing dependency.